src/main/java/org/pkcs11/jacknji11/CKR.java

```diff
@@ -183,9 +183,102 @@ public class CKR {
     public static final long VENDOR_PTK_WLD_LOGIN_CACHE_INCONSISTENT = 0x80002010;
 
     // Thales Luna, from cryptoki_v2.h
+    // Thales Luna, from cryptoki_v2.h
```

Comment:
Am I seeing strange things, or have you duplicated this comment line?

Reply:
Crap, I even checked the diff myself :-). Removed.